ha/setup/ha_setup.py

```diff
@@ -350,6 +391,8 @@ def process(self):
         cluster_secret = Cipher.decrypt(key, cluster_secret.encode('ascii')).decode()
         mgmt_info: dict = self._get_mgmt_vip(machine_id, cluster_id)
         s3_instances = ConfigCmd.get_s3_instance(machine_id)
+        # fetch all nodes stonith config
+        all_nodes_stonith_config: dict = ConfigCmd.get_stonith_config()
```

Comment:
Don't see logic for checking whether it's hw or vm. The stonith should be configured only for hw.

Reply:
Agree, Done. Added check in configure_stonith function